R/aaa.R

```diff
@@ -0,0 +1,25 @@
+vault_client_object <- R6::R6Class(
```

Comment:
this is an interesting ambiguity - I'm using R6_ prefixes usually because I want to have a free constructor function with the same name.  I will think about this